### PR TITLE
add reference to rbenv setup

### PR DESCRIPTION
**Pull request by zawaideh**
The link would be helpful when the first error pops up :)

### CODE REVIEW

File: cmd/brew-bootstrap-rbenv-ruby [hunk 1]
```diff
@@ -29,7 +29,8 @@ trap "cleanup" EXIT
 BASE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
 
 if ! which rbenv &>/dev/null; then
-  abort "Error: you need to install 'rbenv' and put it in your PATH!"
+  warn "Error: you need to install 'rbenv' and put it in your PATH!"
+  abort "https://github.com/sstephenson/rbenv#basic-github-checkout"
```
Comment (mikemcquaid) [inline cmd/brew-bootstrap-rbenv-ruby]: Mind changing this link to `https://github.com/rbenv/rbenv#homebrew-on-mac-os-x`?

**Comment by mikemcquaid** [comment]
One tweak but otherwise :+1: and thanks for doing this :bow:

**Comment by zawaideh** [comment]
How about the link here: 

https://github.com/zawaideh/homebrew-bootstrap/blob/patch-1/cmd/brew-bootstrap-rbenv-ruby#L55

**Comment by mikemcquaid** [comment]
Perfect, thanks!

**Pull request merged**